README.md

```diff
@@ -2096,6 +2096,31 @@ val a: Unit < Routes =
 
 For further examples, Kyo's [example ledger service](https://github.com/getkyo/kyo/tree/main/kyo-examples/jvm/src/main/scala/kyo/examples/ledger) provides practical applications of these concepts.
 
+### ZIOs: Integration with ZIO
+
+The `ZIOs` effect provides seamless integration between Kyo and the ZIO library. The effect is designed to enable gradual adoption of Kyo within a ZIO codebase. It allows you to use ZIO effects within Kyo computations and vice-versa, leveraging the power and expressiveness of both solutions.
```

Comment:
> and vice-versa

We don't yet expose a way to run `Fibers` in the `ZIOs` loop, so I would prefer not to mention this yet. Though it should be safe to port the implementation I built for zio-test to a public method.

Reply:
I'm not sure I follow. The effect integrates fibers from both libraries: https://github.com/getkyo/kyo/blob/27eab354698b290b440c0bba37141747d631aed3/kyo-zio/shared/src/main/scala/kyo/zios.scala#L33

Reply:
I understand that the effect uses fibers. ~~However, since the `run` function does not accept `Fibers`, how can I run this: `val a: Int < Fibers & ZIOs` to a: `Task[Int]`?~~

I realize now that the run function does accept `Fibers` - this is non obvious unless you look closely at the opaque type.

```
ZIOs.run(Fibers.sleep(10.seconds))
```

Might be a good opportunity to add examples 🙂 Optionally we can change the signature to accept `ZIOs & Fibers` - should be semantically equivalent